### ISSUE TITLE
Fatal Error with latest commit: Undefined constant "STDOUT"

### ISSUE THREAD

**Issue by TQPS** [Sarjuuk/aowow#396]
**Describe the bug and how to reproduce it**
Just completed a git pull from the last 2 weeks and now I cannot access the main page; getting the following error:
```
Fatal error: Uncaught Error: Undefined constant "STDOUT" in C:\Servers\WOTLK.bots\_Server\apache\htdocs\database\includes\kernel.php:19 Stack trace: #0 C:\Servers\WOTLK.bots\_Server\apache\htdocs\database\includes\shared.php(24): require_once() #1 C:\Servers\WOTLK.bots\_Server\apache\htdocs\database\index.php(3): require('C:\\Servers\\WOTL...') #2 {main} thrown in C:\Servers\WOTLK.bots\_Server\apache\htdocs\database\includes\kernel.php on line 19
```

Note: For context, aowow is sitting in the "database" folder.

**Screenshots**
N/A

**System:**
 - OS: Win10
 - PHP version: PHP 8.2.5 (cli) (built: Apr 12 2023 08:43:43) (ZTS Visual C++ 2019 x64)
 - revision used: 494328cf5339e6e30976895b530129f5c643c1dc
 - Browser (in case of JavaScript / display errors): Firefox latest (124.0.1 (64-bit))
 - AzerothCore: no

**Comment by Sarjuuk** [comment]
should be fixed in 6249ac6b2afdb1393a0dfd55c03faee20ae2af33

**Comment by TQPS** [comment]
Confirmed. 

Thank you as always.
